Makefile

```diff
@@ -170,9 +170,7 @@ ifdef COMPILE_WITH_TSAN
 	PLATFORM_CXXFLAGS += -fsanitize=thread -fPIC -DROCKSDB_TSAN_RUN
         # Turn off -pg when enabling TSAN testing, because that induces
         # a link failure.  TODO: find the root cause
-	pg =
-else
-	pg = -pg
+	PROFILING_FLAGS =
```

Comment:
build_detect_platform sets PROFILING_FLAGS using a compiler test to detect support for -pg.

The intent here is to maintain the existing behaviour of disabling profiling when building with TSAN.

Aiming for work as before but dynamically disable profiling support on Alpine because it builds everything with -pie which -pg doesn't like.